bgpd/bgp_zebra.c

```diff
@@ -1457,14 +1457,13 @@ void bgp_zebra_announce(struct bgp_dest *dest, const struct prefix *p,
 			has_valid_sid = 1;
```

Comment:
has_valid_sid does not seem relevant anymore.  It should be removed.  The only other place it is used is in a debug statement below and we can just check the flag on the nh to see if it is the right type

Reply:
Yes, this variable doesn't need anymore. So, I deleted it.